### DISCUSSION TITLE
Block editor open in editor view

### DISCUSSION THREAD

**Discussion by futurewebsites** [area17/twill#1184] [General]
I have a created an app.blade.php file which contains a header content area and footer, I yeild('content') in the content area but whenever I add a new block rather then each block being rendered in the content area the entire view is repeated i.e 2 complete pages with one block in each - is this correct behaviour or have I done something wrong? Please note this isn' the preview link its the actual visual bock editor view where you drag and drop blocks of content to the right hand window

**Comment by ifox** [comment]
Hi @futurewebsites, the layout for an individual block should not include your overall page layout. See: https://github.com/area17/twill/blob/030554dd02ae36d35381b55fc73d1ff4996ddcd5/config/blocks.php#L13 

What you can do is to create a core layout for your assets. Then an app layout for your header/footer and a layout for your blocks, which both extend your core layout.
